src/client.cpp

```diff
@@ -524,6 +529,17 @@ static int _connect_check(int ifd) {
     return rc;
 }
 
+#ifdef NO_TIMERS
+void* thread_sleep(void* args)
+{
+    // struct itimerval *temp_timer = (struct itimerval *) args;
+    log_msg("Start sleep for %d seconds", (int)((struct itimerval *)args)->it_value.tv_sec);
+    sleep(((struct itimerval *)args)->it_value.tv_sec);
```

Comment:
Microseconds portion missing. `set_client_timer` handles this portion with the line
```
 timer->it_value.tv_usec =
        (g_pApp->m_const_params.cooldown_msec + g_pApp->m_const_params.warmup_msec) % 1000;
```

Reply:
Ah, missed that! Adding it